packages/web-client/app/components/card-pay/account-display/index.hbs

```diff
@@ -0,0 +1,66 @@
+<div class="account-display" ...attributes>
+  {{#if @balance}}
+    {{#if @isComplete}}
+      {{svg-jar @icon class="account-display__icon" role="presentation"}}
+      <div>
+        <div class="account-display__title-lg" data-test-account-title>
+          {{format-token-amount @balance}} {{uppercase @symbol}}.CPXD
+        </div>
+      </div>
+    {{else}}
+      {{svg-jar @icon class="account-display__icon-sm" role="presentation"}}
+      <div>
+        <div class="account-display__title-sm" data-test-account-title>{{uppercase @symbol}}.CPXD</div>
```

Comment:
If this is intended as generic AccountDisplay, perhaps it should not always have the `.CPXD` suffix.

Reply:
Removed these, also because it's not decided yet if we want to show CPXD everywhere or in some places or not